project_event/models/activity_template.py

```diff
@@ -2,7 +2,7 @@
 # © 2018 Savoir-faire Linux
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
```

Comment:
@rimbendhaou : " # coding: utf-8 -*- " remove this line .No more require in the version 11.0